source/_integrations/aten_pe.markdown

```diff
@@ -0,0 +1,59 @@
+---
+title: "ATEN Rack PDU"
+description: "Instructions on how to integrate ATEN Rack PDUs into Home Assistant."
+ha_category:
+  - Switch
+ha_release: 0.103
+---
+
+The `aten_pe` integration lets you control [ATEN Rack PDUs](https://www.aten.com/eu/en/products/energy-intelligence-pduupsracks/rack-pdu/) from Home Assistant.
+
+In order to use it, SNMP must be enabled on your PDU. For an outlet to appear in Home Assistant, its name must have been configured.
+
+It is recommended to use SNMPv3 to protect your credentials from eavesdropping.
+
+Tested devices:
+  * [PE8324G](https://www.aten.com/eu/en/products/energy-intelligence-pduupsracks/rack-pdu/pe8324/)
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+switch:
+  - platform: aten_pe
+    host: mypdu.lan.
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Secondly, I notice the period (`.`) at the end. Is that needed? It might need some additional notes on that in that case.

Reply:
While `host` is optional, the only sane default can be `localhost`, which will in turn be used only rarely with this integration, i.e. only when port forwarding is involved.

A period at the end of a host marks a FQDN. This is standard syntax in DNS. It allows the resolver to skip iterating over search domains. It's not required for or special to this integration. I can remove it though, if you think it causes confusion.

Reply:
If the host is optional in the integration, it need to be removed from the initial example. Optional configuration options are not allowed in the initial and minimal example.

I understand the `.`, however, consider a non-technical user, as this is documentation.

Considering your comment, I'm kinda thinking `host` should not be optional.

Reply:
I'm going to make host non-optional and use the default ip-address of the device in the example.

Reply:
Btw., auth_key and priv_key are also optional (even though good practice). I'll remove them.